resources/js/visual-editor/site-editor/styles/style-book-canvas.css

```
@@ -0,0 +1,101 @@
+
+.ap-site-editor__style-book-variations {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 0.5rem;
+    padding: 0.5rem 0;
+}
+
+.ap-site-editor__style-book-variation {
+    appearance: none;
+    background: transparent;
+    border: 1px solid color-mix(in srgb, currentColor 25%, transparent);
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)